schema/policy.go

```diff
@@ -413,6 +430,59 @@ func (p *Policy) AddTenantToFilter(action string, tenant Tenant) {
 	p.actionTenantFilter[action] = append(p.actionTenantFilter[action], tenant)
 }
 
+// AddPropertyConditionFilter adds property based filter for action
+func (p *Policy) AddPropertyConditionFilter(action string, match map[string]interface{}) {
+	p.actionPropertyConditionFilter[action] = append(p.actionPropertyConditionFilter[action], match)
+}
+
+// ApplyPropertyConditionFilter applies filter based on Property
+func (p *Policy) ApplyPropertyConditionFilter(action string, data map[string]interface{}, next map[string]interface{}) error {
```

Comment:
I've changed var name and added comment around here.


// ApplyPropertyConditionFilter applies filter based on Property
// You need to pass candidate update value in updateCandidateData on update API, so
// that we can limit allowed update value.
// Let's say we would like to only allow to update from ACTIVE to ERROR on an API.
// We can define this policy like this.
//
//   - action: 'update'
//     condition:
//     - property:
//         status:
//            ACTIVE: ERROR
//     effect: allow
//     id: member
//     principal: Member
//
// This policy check error in case of followings
// - Original value isn't ACTIVE
// - Update candidate value isn't ERROR